server/utils/tasks/listTable.ts

```diff
@@ -93,13 +147,44 @@ const unallocatedTableRows = (tasks: Array<Task>): Array<TableRow> => {
   return rows
 }
 
-const tasksTableRows = (tasks: Array<Task>, allocatedFilter: string): Array<TableRow> => {
-  const rows: Array<TableRow> =
-    allocatedFilter === 'allocated' ? allocatedTableRows(tasks) : unallocatedTableRows(tasks)
+export const completedAtDateCell = (item: Task): TableCell => ({
+  text: item.outcomeRecordedAt ? DateFormats.isoDateTimeToUIDateTime(item.outcomeRecordedAt) : '',
+})
+
+export const completedByCell = (task: Task): TableCell => ({
+  text: task.allocatedToStaffMember?.name,
```

Comment:
If the `task` doesn't have an `allocatedToStaffMember` property the `text` property in the returned object will be `undefined`. It's better to return an empty string